src/_ffab-hover-active.scss

```diff
@@ -0,0 +1,24 @@
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+// Hover / Active State (ffab)
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+
+$ffab-has-hover-active-state: true !default;
+$ffab-hover-active-transition-speed: 150ms !default;
+$ffab-hover-value: -1px !default;
+$ffab-active-value: 1px !default;
+
+@mixin ffab-hover-active {
+  @if $ffab-has-hover-active-state {
+    .button:not(.disabled):not(.no-hover-active) {
```

Comment:
This is to manually disable the active-hover feature on an individual component. It feels valuable if there is an isolated situation were you don't want to include the feature. I'd like to see your response to my first comment to decide how we want to update this code.